### ISSUE TITLE
Issue with CFRU gen 6 EXP Share

### ISSUE THREAD

**Issue by SolessTheScientist** [haven1433/HexManiacAdvance#245]
I've been trying to use the CFRU flag for the gen 6 Exp share but it doesn't give the rest of my team exp. I've tried doing it with items, events, etc. but it just doesn't work🤷‍♂️. Is it even 0x0906? Is it even in the CFRU build???

**Comment by haven1433** [comment]
You are correct, the EXP share flag is not turned on in the HUBOL patch. If you want to use the EXP share, you'll need to build / use the DPE and CFRU yourself.
